dask/array/core.py

```diff
@@ -976,6 +976,9 @@ def __new__(cls, dask, name, chunks, dtype, shape=None):
     def __getnewargs__(self):
         return (self.dask, self.name, self.chunks, self.dtype)
 
+    def __reduce__(self):
+        return (Array, (self.dask, self.name, self.chunks, self.dtype))
```

Comment:
`__reduce__` supersedes all other pickling methods. You can remove `__getnewargs__`.